### PR TITLE
.gitignore: Ignore editor and other temp files

### PR DESCRIPTION
**Pull request by Monal5031**
Update .gitignore to ignore .idea/
dir .DS_Store and static/acitivity-data.json

Closes https://github.com/coala/community/issues/46

### CODE REVIEW

**Comment by Monal5031** [comment]
Old habit to write descriptive things 😅 Fixed it though @prnvdixit

File: .gitignore [hunk 1]
```diff
@@ -103,3 +103,7 @@ ENV/
 _site/
```
Comment (jayvdb) [inline .gitignore]: This should be the last set, as they are specific to this repository.

.idea and other common problems should appear earlier.

Comment (Monal5031) [reply .gitignore]: Done. 😄

**Comment by jayvdb** [comment]
ack https://github.com/coala/community/pull/47/commits/c204dac86861f67bc22dd4fc01e0e72d35064f46

**Comment by jayvdb** [comment]
unack https://github.com/coala/community/pull/47/commits/2e627b685c6f1d175f9845c383d1a9ac2c554b92

**Comment by jayvdb** [comment]
This wont be necessary after my other patches

**Comment by Monal5031** [comment]
@jayvdb what about the `.idea` dir and `.DS_Store` file 😕 that aren't included in your PRs

**Comment by jayvdb** [comment]
@Monal5031 , common gitignore entries can be done using `moban`.  Setting that up is my next project after getting the GCI app here working.

**Comment by Monal5031** [comment]
Closing this as its not needed anymore.

**Pull request closed without merging**